build/appveyor.sh

```diff
@@ -3,17 +3,20 @@
 # Build script run by Appveyor. Might eventually be less
 # single-purpose, but let's get coverage going ASAP...
 
-cd $(dirname $0)/..
```

Comment:
Doesn't removing this mean that this script is now sensitive to `pwd`? (For the `dotnet` invocations below, e.g.)